Gemfile

```diff
@@ -4,7 +4,7 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '4.1.5'
+gem 'rails', '4.1.14.1'
```

Comment:
Isn't this going to break nightlies? https://www.softwarecollections.org/repos/rhscl/rh-ror41/epel-7-x86_64/ contains 4.1.5 & it doesn't look like tfm SCL contains Rails